src/analyze/analyze.c

```diff
@@ -1323,7 +1323,10 @@ static int cat_config(int argc, char *argv[], void *userdata) {
 
         STRV_FOREACH(arg, argv + 1) {
                 if (arg != argv + 1)
-                        puts("");
+                        printf("%s%*s%s\n\n",
+                               ansi_underline(),
+                               columns(), "",
+                               ansi_normal());
```

Comment:
might be something to split out into a new function that tweaks the logic a bit: when underlining is not available it should just output a double newline without any spaces